libs/barista-components/alert/src/alert.ts

```diff
@@ -29,12 +30,27 @@ import {
     role: 'alert',
     class: 'dt-alert',
     '[class.dt-alert-warning]': 'severity === "warning"',
+    '[class.dt-alert-info]': 'severity === "info"',
+    '[class.dt-alert-success]': 'severity === "success"',
   },
   preserveWhitespaces: false,
   changeDetection: ChangeDetectionStrategy.OnPush,
   encapsulation: ViewEncapsulation.Emulated,
 })
 export class DtAlert {
   /** The severity type of the alert. */
-  @Input() severity: 'error' | 'warning' = 'error';
+  @Input() severity: 'error' | 'warning' | 'info' | 'success' = 'error';
+
+  get severityIcon(): DtIconType {
```

Comment:
It could be stored as a map and used as
```ts
return severityToIconType[this.severity];
```
and the method could be named as
```ts
  get severityIconType(): DtIconType {
```

Reply:
👍 for adding a map but will leave `severityIcon`